scripts/merge_material_libraries.py

```diff
@@ -0,0 +1,48 @@
+from pyne.material import MaterialLibrary
+
+"""
+
+This script allows to merge multiple PyNE material hdf5 file into a single one.
```

Comment:
I wouldn’t technically call this a script. I like the capability of the `merge_material_library` function a lot - so much so I think it should become part of PyNE :) But here I would expect to see a 

```python
if __name__ == “__main__”:
```

somewhere in here with options for `datapaths` and `nucpaths` along with an option for the output filename etc.